ci/scenarios/kuttl.yml

```diff
@@ -22,3 +22,6 @@ post_install_operators_kuttl_from_operator:
         deploy_watcher_service_extra_vars | default({}) |
         combine({ 'watcher_repo': watcher_repo })
       }}
+
+# Enable observability operator
+cifmw_deploy_obs: true
```

Comment:
cifmw_deploy_obs var is used in ci-framework to deploy obs: https://github.com/openstack-k8s-operators/ci-framework/blob/ad6505a6417e9ab40ad91542ba86eb2eaae26e46/playbooks/02-infra.yml#L85 . It is integrated with deploy_edpm.yaml playbook used in upstream EDPM job and also in architecture repo.

Sounds good to use this var.
And, I think
we need to drop https://github.com/openstack-k8s-operators/watcher-operator/blob/main/ci/playbooks/deploy_cluster_observability_operator.yaml and https://github.com/openstack-k8s-operators/watcher-operator/blob/main/ci/scenarios/edpm.yml#L7 in favor of above.

Reply:
As discussed on slack, change related to cifmw_deploy_obs will come in seperate pr. It is not blocking the existing one. It is good to merge!